app/scrapers/exalead.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Exalead(Scraper):
+    """Scrapper class for Exalead"""
```

Comment:
Typo `Scrapper` => `Scraper`

Reply:
@sudheesh001 could you please review again ??